manifests/overlays/prod/secrets/clusters/api.aasthana.dev.datahub.redhat.com.enc.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: cluster-apps.ocp.prod.psi.redhat.com
+    labels:
+        argocd.argoproj.io/secret-type: cluster
+    annotations:
+        managed-by: argocd.argoproj.io
+type: Opaque
+stringData:
+    name: api.aasthana.dev.datahub.redhat.com
+    config:
```

Comment:
I think you might be forgetting the `|` for multi line text, after `config:` see how the encrypted version looks for the other files for reference